src/admin/components/Routes.tsx

```diff
@@ -12,6 +12,7 @@ import StayLoggedIn from './modals/StayLoggedIn';
 import Versions from './views/Versions';
 import Version from './views/Version';
 import { DocumentInfoProvider } from './utilities/DocumentInfo';
+import { getSanitizedLogoutRoutes } from './elements/Logout';
```

Comment:
Should we sanitize the logout route and button in `/src/config/sanitize` instead of having to do it in multiple places? Note, you can also add _defaults_ to the `src/config/defaults` file, and those will be automatically merged in. So I think that's what I'd probably do. Load in defaults, and then there's no need to sanitize manually at all.